packages/preview/exam-cn/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "exam-cn"

```

Comment:
This name is still too descriptive. I would recommend adding an adjective in front to make it less generic.